### PR TITLE
Interface thoughts - first features

### PR DESCRIPTION
**Pull request by bruyeret-kitware**
- OVERALL FIXES: 1 and 2
- FILE NAVIGATOR: 3 to 7 and some elements of 8
- UPLOADING DATA: 11

### CODE REVIEW

File: src/views/dataset/DatasetInfo.vue [hunk 1]
```diff
@@ -285,21 +285,21 @@ export default class DatasetInfo extends Vue {
       return;
     }
 
-    const date = new Date();
     try {
+      const dataset = this.store.dataset;
+      if (dataset === null) {
+        throw new Error("dataset was null when it shouldn't be");
```
Comment (luciemac) [inline src/views/dataset/DatasetInfo.vue:291]: I guess you should put a simpler error message: "Dataset not set"

Comment (bruyeret-kitware) [reply src/views/dataset/DatasetInfo.vue:291]: That some code I only moved around
I'll change both error message for "Dataset not set" and "Config not set"

File: src/App.vue [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
-      <v-btn @click.stop="toggleRightPanel('snapshotPanel')" id="snapshotButton"
-        >Snapshot</v-btn
-      >
-      <user-menu />
-      <v-btn @click.stop="toggleRightPanel('annotationPanel')"
-        >Browse Annotations</v-btn
+      <v-btn
+        color="primary"
+        class="ml-4"
+        :to="{ name: 'newdataset' }"
```
Comment (luciemac) [inline src/App.vue:12]: newdataset -> newDataset?

**Comment by luciemac** [comment]
LGTM then

File: src/App.vue (hunk 2 above)
Comment (bruyeret-kitware) [reply src/App.vue:12]: This is the name of the route I took from src/views/dataset/index.ts

```
path: "/new",
name: "newdataset",
component: NewDataset,
```

```
path: "/import",
name: "importdataset",
component: ImportDataset,
```

Should I refactor "newdataset" and "importdataset" ?

Comment (luciemac) [reply src/App.vue:12]: Ohhhh I see.
No you don't need to change anything!

**Pull request merged**